src/blockdev.rs

```diff
@@ -300,7 +300,16 @@ impl PartTableKpartx {
 
 impl PartTable for PartTableKpartx {
     fn reread(&mut self) -> Result<()> {
-        self.run_kpartx("-u")
+        let delay = 1;
+        for _ in 0..5 {
```

Comment:
I think this is essentially https://bugzilla.redhat.com/show_bug.cgi?id=1915965. And the recommendation from the SME there is indeed to just keep running `kpartx -u` if we want to be sure that it ran without falling back to another `udevadm settle`.

Probably worth linking to that RHBZ somewhere around here?

Reply:
Also would be good to add a e.g. `Related: https://bugzilla.redhat.com/show_bug.cgi?id=1974411` in the commit message.